Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -70,13 +70,21 @@
             "name": "WebAppKind",
             "type": "pickList",
             "label": "Web app kind",
-            "defaultValue": "app",
+            "defaultValue": "webApp",
             "required": false,
-            "visibleRule": "WebAppName = invalidwebappname",
+            "options": {
+                "webApp": "Web App",
```

Comment:
Isn't the value that the REST returns "app" instead of "Webapp"? It will be better to have the REST returned values here so that we can pass this to any data source input later if required.

Reply:
Can you also check the Ibiza CI/CD flow where the templates are dynamically created. They might still be dependent on the default app kinds returned by the REST. I am adding @sachinma to review the changes.

Reply:
will this also impact existing RDs? Existing RDs are likely have "app", "applinux" persisted for this input. Will that impact your visibility rules?

Reply:
+1. We are putting app and applinux etc from Ibiza CD. I would also prefer to keep the values old - and just make it visibile with a friendly name here. Can you share the background for exposing this now/